internal/injector/labels.go

```diff
@@ -7,18 +7,29 @@ package injector
 
 import (
 	"strings"
+	"sync"
 
+	"github.com/DataDog/orchestrion/internal/log"
 	"github.com/dave/dst"
 )
 
 const (
-	ddIgnore = "//dd:ignore"
+	ddIgnore          = "//dd:ignore"
+	orchestrionIgnore = "//orchestrion:ignore"
 )
 
-// ddIgnored returns true if the node is prefixed by a `//dd:ignore` directive.
-func ddIgnored(node dst.Node) bool {
+var warnOnce sync.Once
+
+// isIgnored returns true if the node is prefixed by an `//orchestrion:ignore` (or the legacy `//dd:ignore`) directive.
+func isIgnored(node dst.Node) bool {
 	for _, cmt := range node.Decorations().Start.All() {
+		if cmt == orchestrionIgnore || strings.HasPrefix(cmt, orchestrionIgnore+" ") {
```

Comment:
Why not directly call strings.HasPrefix without the spacr at the end? Am I missing something?

Reply:
Because `//orchestrion:ignoreItAllPlease` is not `//orchestrion:ignore`.